zap/backends/postgresql.py

```diff
@@ -28,10 +28,13 @@ def _psql(self, *command):
         ''' Run a command via psql as the postgres user '''
         if self.debug:
             sys.stderr.write('psql -c "' + ' '.join(command) + '"\n')
-        base_command = ['sudo', '-u', 'postgres', 'psql', '-c']
+        port_string = '{port}'.format(port=self.port)
+        base_command = ['sudo', '-u', 'postgres',
+                        'psql', '-p', port_string, '-c']
```

Comment:
I was testing with DJ and a DATABASE_URL env variable, so I'm guessing not.

Will add a test for an empty port string, if that suffices, not sure if you want a more elegant way to construct the command string at this point.